### DISCUSSION TITLE
AVRCP Browsing support

### DISCUSSION THREAD

**Discussion by bitwisesam** [pschatzmann/ESP32-A2DP#156] [Q&A] [answered]
Hi! Great library, very easy to use. I have only one request: is it possible to add support for AVRCP Browsing in A2DP sink?

**Comment by pschatzmann** [comment]
I haven't found any information that would indicate that the ESP32 is supporting this. 
Did you ?

**Comment by bitwisesam** [comment]
I am not sure. There is AVRC feature bit mask called ESP_AVRC_FEAT_BROWSE, which would indicate that AVRC browsing channel can be enabled. But I couldn't find any functions that would use this enum.

From ESP-IDF Programming Guide:
> enum esp_avrc_features_t
> 
>     AVRC feature bit mask.
> 
>     Values:
> 
>     ESP_AVRC_FEAT_RCTG = 0x0001
> 
>         remote control target
> 
>     ESP_AVRC_FEAT_RCCT = 0x0002
> 
>         remote control controller
> 
>     ESP_AVRC_FEAT_VENDOR = 0x0008
> 
>         remote control vendor dependent commands
> 
>     ESP_AVRC_FEAT_BROWSE = 0x0010
> 
>         use browsing channel
> 
>     ESP_AVRC_FEAT_META_DATA = 0x0040
> 
>         remote control metadata transfer command/response
> 
>     ESP_AVRC_FEAT_ADV_CTRL = 0x0200
> 
>         remote control advanced control commmand/response
>

**Comment by pschatzmann** [comment]
Yes - that's where I got stuck as well...

**Comment by bitwisesam** [comment]
I understand. Thank you for your reply. I will try to make an inquiry at Espressif.

**Comment by gregscullard** [comment, marked as answer]
Looks like it isn't yet supported yet, feature request created by OP is in progress https://github.com/espressif/esp-idf/issues/8385
